open-apparel-registry/open-apparel-registry
# Support Uploading Excel Files

## Overview

Adds support for uploading Excel files. We support both `.xls` and `.xlsx` files. Only the first sheet of the file is read, and is expected to be in the same format as the CSVs we expect.

Connects #484

## Testing Instructions

* Check out this branch and try to upload an Excel file (now there are two sample files in the Facility Lists directory)
  - [x] Ensure `.xls` upload works
  - [x] Ensure `.xlsx` upload works
* Download the CSV template from the site and open in Excel or Numbers or Open Office
* Add some rows. Save the file as an Excel file.
* Upload the new file.
  - [x] Ensure the upload works

## Checklist
